io.fossbr.WhatsAppQT.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.fossbr.WhatsAppQT</id>
```

Comment:
`fossbr.io` doesn't resolve. That mean this id is likely not correct.

Reply:
I don't understand this application id rules, if my application is hosted on gitlab , the id is com.gitlab.fossbr.whatsappqt ?

Reply:
that would work. see https://github.com/flathub/flathub/wiki/App-Requirements#application-id

(you didn't acknowledge reading it when submitting)

Reply:
According to the docs, shouldn't it be `io.gitlab.fossbr.WhatsAppQT` instead of `com.gitlab...`?

Reply:
either is acceptable.